main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -15,10 +15,166 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
 
+import ca.uwaterloo.flix.language.ast.{SourceLocation, Symbol, Type}
```

Comment:
Ensure that copyright headers of the files you wrote (e.g. this one) carry your name.

Reply:
Also please add yourself to `AUTHORS.md`.